dpdd.tex

```diff
@@ -211,7 +212,7 @@ \section{General Considerations}
 
 Most LSST data products will consist of images and/or catalogs. The catalogs will be stored and offered to the users as \emph{relational databases} which they will be able to query. This approach was shown to work well by prior large surveys, for example the Sloan Digital Sky Survey (SDSS).
 
-Different data products will generally be stored in different databases. For example, Prompt data products will be stored in a \emph{Level 1 (Prompt) database}. Data Release ``universal cadence'' products will be stored in a \emph{Data Release database}. The products for special programs may be stored in many different databases, depending on the nature of the program.
+Different data products will generally be stored in different databases. For example, Prompt data products will \oldtext{be}\newtext{include catalogs} stored in a \oldtext{\emph{Level 1 (Prompt) database}} \newtext{Prompt Products Database}. Data Release ``universal cadence'' products will be stored in a \emph{Data Release database}. The products for special programs may be stored in many different databases, depending on the nature of the program.
```

Comment:
It makes sense to note that not all prompt products are in the PPDB, but the same surely applies to DR products too? — not all DR products are in a “Data Release database”.

(Also, you could use your new `\DR{}` command here. 😄 )